tensorflow_lite_support/cc/task/README.md

```diff
@@ -378,6 +378,63 @@ SegmentationResult result = result_or.value();
 
 A CLI demo tool is available [here][4] for easily trying out this API.
 
+
+### Landmark Detector
+
+`LandmarkDetector` accepts [MoveNet](https://tfhub.dev/google/movenet/singlepose/lightning/4) model (with optional TFLite Model Metadata) that 
+conforms to the following spec:
+
+Input tensor (type: `kTfLiteUInt8`):
+
+   - image input of size `[batch x height x width x channels]`.
+   - batch inference is not supported (`batch` is required to be 1).
+   - only RGB inputs are supported (`channels` is required to be 3).
+   - `NormalizationOptions` is not required
+
+Output tensor (type: `kTfLiteFloat32`):
+
+   - one output tensor with 4 dimensions `[1 x 1 x num_keypoints x 3]`,
+     where, `num_keypoints` is required to be 17.
```

Comment:
Remove "where, `num_keypoints` is required to be 17", and explain what `3` stands for (i.e. keypoints position + score)?

Reply:
ok